include/picongpu/plugins/openPMD/openPMDWriter.hpp

```diff
@@ -824,9 +825,8 @@ Please pick either of the following:
                     meta::ForEach<FileCheckpointParticles, CopySpeciesToHost<bmpl::_1>> copySpeciesToHost;
                     copySpeciesToHost();
                     lastSpeciesSyncStep = currentStep;
-
-                    dc.releaseData(MallocMCBuffer<DeviceHeap>::getName());
```

Comment:
It is better to keep this call and guard it. This will simplify refactoring where this function becomes important.